README.md

```diff
@@ -30,7 +30,9 @@ const accountCreator = new AccountCreator({
 })
 
 // Open a popup window prompting the user to create an account.
-const creationResult = await accountCreator.createAccount()
+const creationResult = await accountCreator.createAccount().catch((error) => {
```

Comment:
`try { await promise } catch { .. }`
or
`promise.then(..).catch(..)`

never mix

Reply:
I'll make the change, but out of curiosity, what is wrong with  `await promise.catch(error => {})`. I've been doing that for ages 😄

Reply:
@jnordberg done!

Reply:
Whatever is returned in the .catch block is returned as the creationResult so you end up having to handle the error twice by checking for undefined. TypeScript is good about forcing you to actually do that check but it's a common source of stray undefined vars floating around causing confusing errors.

Besides it's a bit offensive, like putting ketchup on icecream or something :P

Reply:
The only valid use of `await promise.catch` I can think of is where you want to assign a default value if it errors:

```ts
let myVar = await promise.catch((error) => {
  console.warn('Unable to load my thing, using default', error)
  return defaultVar
})
```

But I'd argue its cleaner to write something like

```ts
let myVar
try {
  myVar = await promise
} catch (error) {
  console.warn('Unable to load my thing, using default', error)
  myVar = defaultVar
}
```